python/cuml/cuml/internals/api_decorators.py

```diff
@@ -154,7 +154,9 @@ def wrapper(*args, **kwargs):
                 # Wraps the decorated function, executed at runtime.
 
                 # Accept list/tuple inputs when accelerator is active
-                accept_lists = GlobalSettings().accelerator_active
+                import cuml.accel
```

Comment:
why the late import?

Reply:
I originally had these all as local imports, but later moved them to top-level. Just missed this one.